internal/storage/gcs/bucket.go

```diff
@@ -32,6 +32,9 @@ const (
 	NonHierarchical
```

Comment:
Looks like this const is used to define bucketType enum values. We should keep ReqIdField as a seperate constant.

Reply:
done